### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Abidoyinsola1**
A clone of Youtube page

### CODE REVIEW

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🚫

Please make sure that your PR and README (this one has a wrong heading) are professional before submitting a CRR (Code Review Request).

Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/). Then apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

After implementing the required changes submit another CRR.

You can reach out to me in slack as **srodrig**, happy coding! :wave:

**Comment by addod19** [comment]
> _No description provided._

Let us edit the PR with some title and description. For instance, title => youtube clone, description => CSS flex and float for position elements, sans-serif for font-family or something interesting

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          </div>
+        </div>
+      </div>
+
+      <!-- Sidebar -->
+      <div class="sidebar">
+        <div class="next-title">
+          <p>Up Next</p>
+          <p class="autoplay">Autoplay <i class="fas fa-toggle-on"></i></p>
+        </div>
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<aside>``` instead of ```< div class="sidebar">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Abidoyinsola1, great job :+1: Commit messages clearly but can be improved :slightly_smiling_face:  Let us work to get the project approved by addressing the comments below:
- ``` Nav ```, the elements in the nav needs a little work. Input and social icons
unav
- ``` Main ```, the main content is overlapping with the sidebar
min
- ``` Share and Subscribe Button ```, currently we have a share and subscribe icon, let us change them to button
sb
sb1
- Let us fix the error returned by the w3validator
err
- Let us apply the DRY rule. The following CSS properties have been repeated; ``` display: flex ```, ``` position: absolute ```, ``` display: grid ``` appears more than once. Let us refactor our file
- In our README file, we are repeating ```Built with and Completed with ```, let us remove on and try so add more light to our description :smiley: 

Kindly make these changes and submit another code review :+1: 
_Happy coding_ :man_technologist:
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/) for any clarification or details

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement this change:

> image

- [ ] Kindly fix overlap of the save and share buttons.

### Happy coding! :computer:

## Lucas Mazo

**Comment by HADDADSOHAIB** [comment]
> ### Pull Request under the review by Sohaib Haddad. (TSE)

### Hello @Abidoyinsola1  :handshake: 

### Good job so far :clap:  your project is looking good :medal_sports: 

### there are still some changes requested by the previous TSE to make to get the project approved :point_down: 

- There is still a lot of overlap in your project:
Screenshot from 2020-06-03 11-31-13
Screenshot from 2020-06-03 11-31-21
- Please add descriptive comments to the pull request as requested from the previous TSE:
- your pull request:
Screenshot from 2020-06-03 11-35-07
- you can follow this propositions for a title a description:
Screenshot from 2020-06-03 11-35-19
- The subscribe and share should be buttons, they are not in your project:
- [ ] It has a share button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

- [ ] It has a subscribe button ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))

### Please implement the changes and ask for another code review, good job so far :+1:

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: 
Great work implementing the required changes. I have approved this project but there are some more changes required. Kindly Improve on the following

-  Your search bar needs to be properly positioned at the center of the page.

-  The like and dislike should be buttons but not icons. Try using a CSS  rule to make the cursor to be a pointer.

Great work :clap: kindly do those changes and submit a project completion form. I have approved it because you have made the changes required in the previous reviews. 
on to the next project :fireworks:  Happy coding:computer:

**Pull request merged**